integration_tests/k8s/k8sapi.go

```diff
@@ -104,11 +105,11 @@ func (c k8sClient) ListBlockDeviceClaims() (*apis.BlockDeviceClaimList, error) {
 // NewBDC creates a sample device claim which can be used for
 // claiming a block device.
 func NewBDC(bdcName string) *apis.BlockDeviceClaim {
-	bdcRequirements := apis.DeviceClaimRequirements{
+	bdcRequirements := apis.DeviceClaimResources{
```

Comment:
@prateekpandey14  It would be better to change `bdcRequirements` to `bdcResources`

Reply:
done